kclvm/src/lib.rs

```diff
@@ -78,20 +78,25 @@ pub unsafe extern "C" fn kclvm_cli_main(argc: c_int, argv: *const *const c_char)
     });
     std::panic::set_hook(prev_hook);
 
-    CString::new(match kclvm_cli_main_result {
+    match kclvm_cli_main_result {
         Ok(result) => match result {
-            Ok(()) => "".to_string(),
+            Ok(()) => Box::into_raw(Box::new(ExitCode::SUCCESS)),
             Err(err) => {
                 let backtrace = format!("{}", err.backtrace());
                 if backtrace.is_empty() {
-                    format!("Error: {}\n", err)
+                    print!("Error: {}\n", err);
                 } else {
-                    format!("Error: {}\n\nStack backtrace:\n{}", err, backtrace)
+                    print!("Error: {}\n\nStack backtrace:\n{}", err, backtrace);
```

Comment:
Here's print!() It is possible that something cannot be printed, rust print! is not flush by default.